fluentd/run.sh

```diff
@@ -19,7 +19,7 @@ done
 echo "============================="
 echo "Fluentd logs have been redirected to: $LOGGING_FILE_PATH"
 echo "If you want to print out the logs, use command:"
-echo "oc exec <pod_name> $HOME/utils/logs"
+echo "oc exec <pod_name> logs"
```

Comment:
ok - you only need to use the `oc exec $fluentd_pod -- $command` form if you need to pass command line arguments beginning with `-` to `$command` - in this case, `logs` doesn't have any arguments